hri_safety_sense/srv/KeyString.srv

```diff
@@ -0,0 +1,5 @@
+# KEY VALUE 
```

Comment:
I think this file (and all of the `.srv` files in the `hri_safety_sense` package) can be removed.

Reply:
7de7162